### ISSUE TITLE
AnimationLoader.KoikatsuSunshine.dll is in bug or idk for sure what going on but running in the 1.1.3.4 version 

### ISSUE THREAD

**Issue by MSCKing1** [IllusionMods/AnimationLoader#33]
were stuck in loading when i start on HScene  not freeze just wont start the Hscene 
![Schermafbeelding_2024-06-03_233352](https://github.com/IllusionMods/AnimationLoader/assets/171673708/9f93548e-34f3-4be4-8907-409f28900535)
<img width="1280" alt="image" src="https://github.com/IllusionMods/AnimationLoader/assets/171673708/921ede2b-cfe4-49fa-b262-529d886c3bc3">

**Comment by ManlyMarco** [comment]
Update animation loader and the sideloader animation pack and try again.

**Comment by MSCKing1** [comment]
> Update animation loader and the sideloader animation pack and try again.
I have already reversed twice and it has not resolved. but When I delete the folder called Animation Loader.Koikatsu Sunshine.dll, it solves the problem. so i think something wrong with Animation Loader.Koikatsu Sunshine.dll,

**Comment by ManlyMarco** [comment]
Post your output_log file from game directory.

**Comment by MSCKing1** [comment]
sorry for late respond back 
[output_log.txt](https://github.com/user-attachments/files/15637689/output_log.txt)

**Comment by MSCKing1** [comment]
> Post your output_log file from game directory.

sorry late respond here is output_log 
[output_log.txt](https://github.com/user-attachments/files/15637696/output_log.txt)

**Comment by ManlyMarco** [comment]
> > Post your output_log file from game directory.
> 
> sorry late respond here is output_log [output_log.txt](https://github.com/user-attachments/files/15637696/output_log.txt)

One of the animation zipmods you have in your mods folder is corrupted and causes the plugin to crash.

**Comment by MSCKing1** [comment]
> > > Post your output_log file from game directory.
> > 
> > 
> > sorry late respond here is output_log [output_log.txt](https://github.com/user-attachments/files/15637696/output_log.txt)
> 
> One of the animation zipmods you have in your mods folder is corrupted and causes the plugin to crash.

can u tell me which one ?

**Comment by ManlyMarco** [comment]
No, the error doesn't specify. Update the animation modpack and remove any animations outside of the modpack first.

**Comment by MSCKing1** [comment]
> No, the error doesn't specify. Update the animation modpack and remove any animations outside of the modpack first.

oke

**Comment by MSCKing1** [comment]
> > > Post your output_log file from game directory.
> > 
> > 
> > sorry late respond here is output_log [output_log.txt](https://github.com/user-attachments/files/15637696/output_log.txt)
> 
> One of the animation zipmods you have in your mods folder is corrupted and causes the plugin to crash.
 what i try tips from you dont seems to works crash wil keep come back 
i guess i have to play without animation loader

**Comment by MSCKing1** [comment]
> No, the error doesn't specify. Update the animation modpack and remove any animations outside of the modpack first.
i try from you tips dont works  it happend every time again broke or currputed just idk to me but it give same apear as before problem when i removed outside mod and update animation nothing works i guess i have to play without animation loader

**Comment by IDontHaveIdea** [comment]
Hi,

Delete this file:

UserData\AnimationLoader\Usage\AnimationsUsage.xml

Then check again.
